user-management.yaml

```
@@ -446,6 +460,13 @@ paths:
         - uc4_auth: [] 
       description: "Returns all lecturers, may only be used by administrators"
```

is this still true, or can all fetch users using a list of usernames, similar to above?
```suggestion
      description: ""Without parameter \"usernames\", returns all lecturers, and may only be used by administrators. With \"usernames\" set, returns all lecturers with the given usernames, which can be invoked by all roles.""
```